build-prybar.nix

```diff
@@ -0,0 +1,31 @@
+{ language, buildInputs ? [] }:
+
+{ lib, buildGoModule, fetchFromGitHub, bash, pkg-config }:
+
+buildGoModule rec {
+    name = "prybar-${language}";
+    version = "98148ff";
```

Comment:
Can there be a comment as to where this version comes from? or do the same trick that @cbrewster did on goval: https://github.com/replit/goval/blob/82a66b26713f1449450cb23fafcfea045416cf53/default.nix#L13

if it's temporary, you can use the date in "YYYY-MM-DD" while we figure out a better way to get the correct hash.